### PR TITLE
Update settings.gradle

### PR DESCRIPTION
**Pull request by minhnguyenbhs**
forgot to import controls

### CODE REVIEW

File: controls/src/build.gradle [hunk 1]
```diff
@@ -1,20 +1,51 @@
-/*
- * This file was generated by the Gradle 'init' task.
- *
- * This generated file contains a sample Java library project to get you started.
- * For more details on building Java & JVM projects, please refer to https://docs.gradle.org/8.7/userguide/building_java_projects.html in the Gradle documentation.
- */
+import java.text.SimpleDateFormat
```
Comment (jkleiber) [inline controls/src/build.gradle:1]: Is this necessary? This is only for commit on deploy I think

File: controls/src/build.gradle [hunk 2]
```diff
@@ -1,20 +1,51 @@
...
 	id "com.diffplug.spotless" version "6.24.0"
+}
+
+java {
+	sourceCompatibility = JavaVersion.VERSION_17
+	targetCompatibility = JavaVersion.VERSION_17
+}
+
+// Set this to true to enable desktop support.
+def includeDesktopSupport = true
```
Comment (jkleiber) [inline controls/src/build.gradle:16]: Is this necessary? This is only for WPILib

File: controls/src/build.gradle [hunk 3]
```diff
@@ -1,20 +1,51 @@
...
+tasks.withType(JavaCompile) {
+	options.compilerArgs.add '-XDstringConcat=inline'
+}
+
+project.compileJava.dependsOn(createVersionFile)
+gversion {
+srcDir       = "src/main/java/"
+dateFormat   = "yyyy-MM-dd HH:mm:ss z"
+timeZone     = "America/New_York"
+indent       = "  "
```
Comment (jkleiber) [inline controls/src/build.gradle:42]: I don't think we need this - this is for commit on deploy I think

**Review by jkleiber — COMMENTED** [review]
first pass comments - we mainly just need to remove unused things in the build.gradle

**Comment by minhnguyenbhs** [comment]
I think I removed all of these unused commands in the other gradle edit branch- thank you for your help!

**Pull request closed without merging**